charts/node/templates/service.yaml

```diff
@@ -75,8 +77,10 @@ spec:
     {{- $selectorLabels | nindent 4 }}
     statefulset.kubernetes.io/pod-name: {{ $fullname }}-{{ $i }}
   ports:
+    {{- if $.Values.node.legacyRpcFlags }}
     - port: {{ $.Values.node.perNodeServices.apiService.httpPort | int }}
       name: http-rpc
+    {{- end }}
     - port: {{ $.Values.node.perNodeServices.apiService.wsPort | int }}
       name: websocket-rpc
```

Comment:
Rename this port to `rpc`

Reply:
we should keep old name in case someone uses ingress.